tests/conftest.py

```diff
@@ -18,10 +18,9 @@
 #
 # ------------------------------------------------------------------------------
 """Conftest module for Pytest."""
-import atexit
```

Comment:
anything specific to libp2p tests - that is to say related utility functions and constants - has been moved into the libp2p test folder, specifically the helper module [base.py](https://github.com/valory-xyz/open-aea/blob/10a3e0aa5b58262b944867222764c4f62feb32a7/tests/test_packages/test_connections/test_p2p_libp2p/base.py)